CHANGELOG.md

```diff
@@ -1,11 +1,17 @@
 ## 5.5.1 (YYYY-MM-DD)
 
-### Bug Fixes
+### Enhancements
+* None
 
-* Building with Android App Bundle enabled should now work correctly (#5977).
+### Fixes
```

Comment:
Why not the much more common `Bug Fixes` which is also used by other repos

Reply:
In some cases you can around back and forth if a small "fix" is a bug or not. It will be the same across all repos.